vnc_auto.html

```diff
@@ -151,13 +151,17 @@
         // Set parameters that can be changed on an active connection
         rfb.viewOnly = readQueryVariable('view_only', false);
         rfb.scaleViewport = readQueryVariable('scale', false);
+        if (null === myRFB) {
+            myRFB = rfb;
+        }
```

Comment:
@pawan-idrive This may not be necessary. You can create a `RFB` object in our own javascript by passing the `shared=true`. Refer here. https://novnc.com/noVNC/docs/API.html#syntax. This should leave html completely untouched. Even for `connect` even callback also this should be possible without updating the html.

Reply:
@pawan-idrive Did you try this? Otherwise everything else looks fine.

Reply:
I had tried to look into this but could not spend much time, will see it afterward and will see, if this can be implemented. As of now, I think, we won't be able to.